src/clientdlg.cpp

```diff
@@ -1479,7 +1479,14 @@ void CClientDlg::SetMixerBoardDeco ( const ERecorderState newRecorderState, cons
         }
         else
         {
-            if ( palette().color ( QPalette::Window ) == QColor::fromRgbF ( 0.196078, 0.196078, 0.196078, 1 ) )
+#if QT_VERSION >= QT_VERSION_CHECK( 6, 5, 0 )
+            // for Qt 6.5.0 or later, we use the inbuilt cross platform color scheme picker.
+            if ( QGuiApplication::styleHints()->colorScheme() == Qt::ColorScheme::Dark )
+#else
+            // for earlier versions, check darkmode as proposed in https://www.qt.io/blog/dark-mode-on-windows-11-with-qt-6.5
+            const QPalette defaultPalette;
+            if ( defaultPalette.color ( QPalette::WindowText ).lightness() > defaultPalette.color ( QPalette::Window ).lightness() )
+#endif
             {
                 // Dark mode on macOS/Linux needs a light color
```

Comment:
Of course, this is now "Dark mode on QT before 6.5.0" rather than OS-specific.

Reply:
(Also out of scope, I'd rather we were using named CSS colours rather than magic incantations (what's wrong with "[gainsboro](https://developer.mozilla.org/en-US/docs/Web/CSS/named-color)" or "[black](https://developer.mozilla.org/en-US/docs/Web/CSS/named-color)"?).)

Reply:
They are not magic at all. rgb(x,y,z) or #ffffff are clearly defined color codes. In webdev I usually also don't use keywords as e.g. the hex value is more precise. Also IDEs usually show the color directly:

![color picker in VSCodium](https://github.com/user-attachments/assets/34b42df2-a562-4029-a1d0-b6fc5b648f6a)